test/golden/npm-package/index.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html><html><head>
     <script type="module">
-import * as test from './node_modules/test-dependency/dep.js';</script>
+import * as test from '../test-dependency/dep.js';</script>
```

Comment:
@justinfagnani confirm these changes are okay? This looks like it was caused by https://github.com/Polymer/polymer-build/pull/320 which is getting pulled into the next release

Reply:
this file was being loaded as an app-style request, so the URL should really be `./node_modules/...`